modules/ROOT/pages/dashboard-custom-config.adoc

```diff
@@ -61,18 +61,20 @@ image::dashboard-custom-config-high-level.png[Dashboard Configuration]
 | Tags | A short identifier that can help you locate a dashboard in the custom dashboard list. See xref:dashboards-using.adoc#favorite_dashboard[Setting Favorite Custom Dashboards] for an example.
 | Timezone a|
 
-Most graphs track data over time, so your time options are important. Here are your options:
+Most graphs track data over time, so your time options are important. Your options are:
 
-* Default: The default is the Local Time.
-* Local Browser Time: Time taken from the system time of the computer that the browser is running on.
-* UTC: Coordinated Universal Time standard, which uses the 24 hour cycle (for example, `19:20`) and corresponds to GMT.
-| Graph Tooltip | Options: Default, Shared Crosshair, Shared Tooltip. With Default, the crosshair cursor only appears in the chart that you mouse over. With Shared Crosshair, the crosshair appears on all charts, and with Shared Tooltip, the crosshair and the tooltip appears.
+* *Local browser time*: (Default) Time taken from the system time of the computer the browser is running on.
```

Comment:
I'd put a "that" in front of "the browser" to aid reading. Also I don't know what "Time taken from the system time" means.